modules/react/button/stories/visual-testing/stories_ExternalHyperlink.tsx

```diff
@@ -44,7 +44,11 @@ export const ExternalHyperlinkStates = () => (
         {(props: any) => (
           <Container blue={props.variant === 'inverse'}>
             <Subtext as="span" size="large" color="#000" variant={props.variant}>
-              Here's a <ExternalHyperlink {...props}>Link</ExternalHyperlink> to something
+              Here's a{' '}
```

Comment:
is `{' '}` something we need it?

Reply:
This looks like some prettier shenanigans. I'll remove it.

Reply:
I would not recommend fighting with prettier unless you know for sure your styling is intentional and Prettier is wrong. Then you can use `// prettier-ignore`. Know that Prettier will no longer be run on that line.

I used this once on a data-file that listed out data in a table-like format. The table format made it much easier to understand the data relationship (rows and columns) and Prettier mangled that relationship.

For most things though, leave Prettier to format the files.